tests/unit/plugins/inventory/test_now.py

```diff
@@ -50,21 +50,21 @@ def test_no_query(self, table_client):
         now.fetch_records(table_client, "table_name", None)
 
         table_client.list_records.assert_called_once_with(
-            "table_name", dict(sysparm_display_value=True)
```

Comment:
the tests need to take into account the new option `display_value` and not hard coded value for `sysparm_display_value`